mailgun/route_handlers.py

```diff
@@ -42,162 +42,159 @@ def handle_mailing_list_email_route(request):
     :return:
     '''
     sender = request.POST.get('sender')
-    recipient = request.POST.get('recipient')
+    recipients = set(address.parse_list(request.POST.get('recipient')))
     subject = request.POST.get('subject')
     body_plain = request.POST.get('body-plain', '')
     body_html = request.POST.get('body-html', '')
     message_id = request.POST.get('Message-Id')
-    to_list = address.parse_list(request.POST.get('To'))
-    cc_list = address.parse_list(request.POST.get('Cc'))
+    to_list = request.POST.get('To')
+    cc_list = request.POST.get('Cc')
```

Comment:
Gah...never mind.  Those are getting passed in as original_to_address/original_cc_address, they can totally be left alone.  Sorry for the false alarm.